app/uk/gov/hmrc/plasticpackagingtax/registration/models/genericregistration/PartnerContactDetails.scala

```diff
@@ -25,7 +25,15 @@ case class PartnerContactDetails(
   emailAddress: Option[String] = None,
   phoneNumber: Option[String] = None,
   address: Option[Address] = None
-)
+) {
+
+  def contactName: Option[String] =
```

Comment:
The UI contains headings like 'Enter the email for -contact name-' - hence this helper method for deal with fullName vs firstName / lastName combinations.